packages/mobile/src/invite/saga.ts

```diff
@@ -182,15 +183,32 @@ export function* sendInvite(
     const inviteCode = createInviteCode(temporaryWalletAccount.privateKey)
     const name = yield select(nameSelector)
 
-    const link = yield call(generateInviteLink, inviteCode)
-    const message = i18n.t(
-      amount ? 'sendFlow7:inviteWithEscrowedPayment' : 'sendFlow7:inviteWithoutPayment',
-      {
-        name,
-        amount: amount?.toString(),
-        link,
+    const link = features.ESCROW_WITHOUT_CODE
+      ? VALORA_DOMAIN
+      : yield call(generateInviteLink, inviteCode)
+
+    let messageProp
+    if (features.ESCROW_WITHOUT_CODE && amount) {
```

Comment:
I don't think the `features.ESCROW_WITHOUT_CODE` should affect the string used, we should still use one of the other two.